boot/zephyr/CMakeLists.txt

```diff
@@ -198,7 +198,6 @@ if(CONFIG_MCUBOOT_SERIAL)
 
   zephyr_link_libraries_ifdef(
     CONFIG_TINYCBOR
-    TINYCBOR
     )
```

Comment:
The whole zephyr_link_libraries_ifdef invocatoin should be removed.

Reply:
OK